app/views/shared/_navbar.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="navbar navbar-expand-sm navbar-light navbar-lewagon">
   <%= link_to "#", class: "navbar-brand" do %>
-    <%= image_tag "https://raw.githubusercontent.com/lewagon/fullstack-images/master/uikit/logo.png" %>
+    <%= image_tag "persed_logo.png" %>
     <% end %>
```

Comment:
identation here on line 4 as well 😁 and in link_to we can already put root_path I think (instead of #)